### PR TITLE
move h1 to top, slight revisions

### PR DESCRIPTION
**Pull request by ghost**
Modified headers and titles slightly in order to get the h1 to the top.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
 
 - [Discord Server](https://discord.gg/mpZ9AYD)
 - [Telegram Channel](https://t.me/casperofficialann)
 
-# casper-node
 
-This is the core application for the Casper blockchain.
 
-## Running a validator node from Source
+## Running a Validator casper-node from Source
```
Comment (EdHastingsCasperAssociation) [inline README.md]:
```suggestion
## Running a casper-node from source
```

**Review by EdHastingsCasperAssociation — APPROVED** [review]

**Review by Fraser999 — APPROVED** [review]

**Comment by Fraser999** [comment]
bors merge

**Comment by Fraser999** [comment]
bors merge

**Comment by Fraser999** [comment]
@abebeos - thanks for the PR!

**Pull request merged**